tests/data/cpu_template_helper/fingerprint_INTEL_SKYLAKE_4.14host.json

```diff
@@ -183,7 +183,7 @@
           },
           {
             "register": "ecx",
-            "bitmap": "0b00000000000000001011111111111111"
+            "bitmap": "0b00000000000000001000111111111111"
```

Comment:
it would be good to add which bits we are trying to change and what they are in the commit message. Maybe https://github.com/firecracker-microvm/firecracker/pull/4410#issuecomment-1914597846 has the info that can be added ?

Reply:
Added info in the commit.